packages/x-date-pickers/src/internals/components/PureDateInput.tsx

```diff
@@ -140,6 +140,10 @@ export const PureDateInput = React.forwardRef(function PureDateInput<TInputDate,
     () => ({
       ...InputProps,
       readOnly: true,
+      // explicit `z-index` on input makes sure the input is above label in the stacking context
+      // `filled` and `outlined` variants of `InputLabel` having explicit `z-index: 1` forces an override here
+      // this solves a bug uncovered only in Mobile (touch input) Chrome
+      sx: { zIndex: 1 },
```

Comment:
From the [preview](https://deploy-preview-5651--material-ui-x.netlify.app/x/react-date-pickers/date-picker/), I don't see that the `zIndex` is applied, do I miss something?

Reply:
It is being applied to the `div` wrapping this input.
![Screenshot 2022-08-23 at 11 54 56](https://user-images.githubusercontent.com/4941090/186116533-15240d88-dca0-42e4-9351-9ad75c404311.png)

Reply:
I got a double `z-index`🤔 in https://deploy-preview-5651--material-ui-x.netlify.app/x/react-date-pickers/date-picker/#responsiveness
<img width="1141" alt="image" src="https://user-images.githubusercontent.com/18292247/186118164-18b0efaf-576f-4954-9b0e-89b11a4d2db9.png">

Reply:
Well, that is very peculiar... 🤔 
Which browser is it?

Reply:
FYI, I did replicate the double application of any `sx` prop passed after syncing with origin (the latest commit merging master into this branch).
Investigating.